x-pack/plugins/security_solution/public/siem_migrations/rules/pages/index.tsx

```diff
@@ -34,13 +34,11 @@ export const RulesPage: React.FC<RulesMigrationPageProps> = React.memo(
 
     const { data: ruleMigrationsStatsAll, isLoading: isLoadingMigrationsStats } = useLatestStats();
 
-    const migrationsIds = useMemo(() => {
+    const filnishedRuleMigrationsStats = useMemo(() => {
       if (isLoadingMigrationsStats || !ruleMigrationsStatsAll?.length) {
         return [];
       }
-      return ruleMigrationsStatsAll
-        .filter((migration) => migration.status === 'finished')
-        .map((migration) => migration.id);
+      return ruleMigrationsStatsAll.filter((migration) => migration.status === 'finished');
```

Comment:
consider using the enum:
```suggestion
      return ruleMigrationsStatsAll.filter((migration) => migration.status === SiemMigrationTaskStatus.FINISHED);
```